packages/lib/shared/components/promos/PromoBanners.tsx

```diff
@@ -19,113 +19,34 @@ import { PromoHookIcon } from '../icons/promos/PromoHookIcon'
-    buttonText: 'View pools',
-    buttonLink: '/pools?poolTags=BOOSTED',
-    linkText: 'Learn more',
-    linkURL: 'https://docs.balancer.fi/concepts/explore-available-balancer-pools/boosted-pool.html',
-    linkExternal: true,
-    bgImageActive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-active0',
-    },
-    bgImageInactive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-inactive0',
-    },
-  },
-  {
-    id: 1,
-    icon: <PromoVThreeIcon size={44} />,
-    title: 'Balancer v3 is live and thriving!',
-    description:
-      'A simple, flexible, powerful platform to innovate upon and build the future of AMMs. Battle-tested on-chain since November.',
-    buttonText: 'View pools',
-    buttonLink: 'pools?protocolVersion=3',
-    linkText: 'Learn more',
-    linkURL: 'https://docs.balancer.fi/partner-onboarding/balancer-v3/v3-overview.html',
-    linkExternal: true,
-    bgImageActive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-active1',
-    },
-    bgImageInactive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-inactive1',
-    },
-  },
-  {
-    id: 2,
-    icon: <PromoGyroIcon size={44} />,
-    title: 'Superliquidity, made simple',
-    description:
-      'Next generation Gyroscope pools are now live on Balancer v3. Manage liquidity directly within the Balancer UI.',
-    buttonText: 'View pools',
-    buttonLink: '/pools?protocolVersion=3&poolTypes=GYRO',
-    linkText: 'Learn more',
-    linkURL: 'https://www.gyro.finance/',
-    linkExternal: true,
-    bgImageActive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-active2',
-    },
-    bgImageInactive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-inactive2',
-    },
-  },
-  {
-    id: 3,
-    icon: <PromoHookIcon size={44} />,
-    title: 'StableSurge Hook',
-    description:
-      'A dynamic directional surge swap fee in times of volatility to help defend the peg. LPs get MEV protection and increased fees.',
-    buttonText: 'View pools',
-    buttonLink: '/pools?poolHookTags=HOOKS_STABLESURGE',
-    linkText: 'Learn more',
-    linkURL: 'https://medium.com/balancer-protocol/balancers-stablesurge-hook-09d2eb20f219',
-    linkExternal: true,
-    bgImageActive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-active3',
-    },
-    bgImageInactive: {
-      directory: '/images/promos/promo-banner/',
-      imgName: 'bg-inactive3',
-    },
-  },
-]
+const promoData: (PromoItem & { iconElement: React.ReactNode })[] =
+  PROJECT_CONFIG.promoItems?.map(item => ({
+    ...item,
+    iconElement: getIconElement(item.icon),
+  })) ?? []
 
 export function PromoBanners() {
   const { colorMode } = useColorMode()
-  const [activeIndex, setActiveIndex] = useState(0)
+  const [activeIndex, setActiveIndex] = useState(getRandomInt(0, promoData.length - 1))
```

Comment:
isn't there one promo item more importante than the others?

Reply:
@uiuxxx is ok with this, so i don't think so